impl/substrate.rc.in

```diff
@@ -0,0 +1,32 @@
+#include <windows.h>
+
+VS_VERSION_INFO VERSIONINFO
+FILEVERSION     @VER_FILEVERSION@
+PRODUCTVERSION  @VER_PRODUCTVERSION@
+FILEFLAGSMASK   VS_FFI_FILEFLAGSMASK
+FILEOS          VOS_NT_WINDOWS32
+FILETYPE        VFT_DLL
+FILESUBTYPE     VFT2_UNKNOWN
+BEGIN
+	BLOCK "StringFileInfo"
+	BEGIN
+		BLOCK "040904E4"
+		BEGIN
+			VALUE "CompanyName",      "bad_alloc heavy industries"
+			VALUE "FileDescription",  @VER_FILEDESCRIPTION_STR@
+			VALUE "FileVersion",      @VER_FILEVERSION_STR@
+			VALUE "InternalName",     ""
+			VALUE "LegalCopyright",   "Copyright (c) 2020, Rachel Mant, Aki Van Ness. All rights reserved. SPDX-License-Ref: @license@"
```

Comment:
We've been pretty bad at keeping these notices updated (oops!) but this should probably be made to read `2020-2023` for the copyright years

Reply:
Your license information says 2020, hence why I preserved that information.

Reply:
Aye, it needs fixing too